package.json

```diff
@@ -2,6 +2,7 @@
   "name": "root",
   "private": true,
   "scripts": {
+    "audit-all": "lerna run audit-moderate",
```

Comment:
should this just be `audit`?

`check-all` runs multiple commands (format-check, lint, test, build)

Reply:
does this check the root package.lock.json or just the individiual projects?

if just invidivual projects, consider `npm audit --audit-level=moderate && lerna run audit-moderate`

Reply:
It can't be audit since there's already an npm audit cmd.  lerna run audit-moderate will run the audit-moderate script cmd in each package and each packages audit-moderate action calls npm audit with a level of moderate.